cognite/neat/rules/models/dms/_schema.py

```diff
@@ -60,6 +62,30 @@ class DMSSchema:
         "node": "node_types",
     }
 
+    def _get_mapped_container_from_view(self, view_id: dm.ViewId) -> set[dm.ContainerId]:
+        # index all views, including ones from reference
+        indexed_views = {
+            **{view.as_id(): view for view in self.views},
+            **({view.as_id(): view for view in self.reference.views} if self.reference else {}),
+        }
```

Comment:
Suggest adding this as a cached_property as it is independent of the method input.

Reply:
Nevermind, that will cause issues if more views are added later.